### PR TITLE
Clean up the username for display

### LINKED ISSUE #1438: username shown is prefixed with 'user-'
Because of how we log the user in whenever the user data is requested with `getCredentials` we prefix it with user. This means that in the breadcrumb and in the profile view we show things like `user-admin` and `user-admin@local` which imho is quite ugly and not representative of how the user actually logged into the GUI. I feel like we should display the username that they used to login which is without `user-`

### PR DESCRIPTION
**Pull request by huwshimi**
Replace the internal parts of the username and just show the user facing bits. Fixes #1438.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load the GUI
- in the breadcrumb and userprofile the username should be without "admin-" or "@local"

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -2131,6 +2131,9 @@ YUI.add('juju-gui', function(Y) {
         user = users.jem ||
                controllerUser ||
                users.charmstore;
+        if (user && user.user) {
+          user.user = user.user.replace('user-', '').replace('@local', '');
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: I THINK we want to keep the @local in there.

Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: @urosj Thoughts?

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4607/
Test PASSed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4608/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4609/
Test PASSed.

**Comment by anthonydillon** [comment]
Code :+1: will QA later if none beats me to it.

**Comment by anthonydillon** [comment]
After a clean-all and running GUI in a private window. I username was admin then I logged in and the username remained admin. In the profile and the breadcrumb.

**Comment by hatched** [comment]
@anthonydillon this needs to be qa'd in a real env

**Comment by hatched** [comment]
Thanks huw! :+1:

**Comment by kadams54** [comment]
:+1: from me as well.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by hatched** [comment]
QA :ok:

**Pull request merged**